paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/PaymentDetails.kt

```diff
@@ -102,13 +106,110 @@ internal fun PaymentDetailsListItem(
                     )
                 }
             }
+
+            MenuAndLoader(
+                enabled = enabled,
+                isUpdating = isUpdating,
+                onMenuButtonClick = onMenuButtonClick
+            )
```

Comment:
To align with iOS, should we allow updating payment methods, even if they can't be used?

Reply:
Done